### PR TITLE
Optimize the annotation of worker-size to avoid ambiguity

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/781
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
@sunzhaoyang Please review the modification in Chinese, thx!

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -116,8 +112,8 @@ loaders:
 # Configuration arguments of the sync processing unit.
 syncers:
   global:                            # The configuration name of the processing unit.
-    worker-count: 16                 # The number of threads that replicate binlog events concurrently in the sync processing unit. When multiple instances are migrating data to TiDB at the same time, reduce the value according to the load.
-    batch: 100                       # The number of SQL statements in a transaction batch that the sync processing unit replicates to the downstream database (100 by default).
+    worker-count: 16                 # The number of concurrent threads that read, parse, and apply binlog events in the sync processing unit (16 by default). Adjusting this value has no effect on the upstream, but has a significant effect on the downstream database.
```
Comment (sunzhaoyang) [inline en/task-configuration-file-full.md:115]:
```suggestion
    worker-count: 16                 # The number of concurrent threads that read, parse, and apply binlog that has been transferred to the local(16 by default). Adjusting this value has no effect on the upstream, but has a significant effect on the downstream database.
```

File: zh/task-configuration-file-full.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ loaders:                             # load 处理单元的运行配置参数
 
 syncers:                             # sync 处理单元的运行配置参数
   global:                            # 配置名称
-    worker-count: 16                 # sync 读取和应用已传输 binlog event 的并发线程数量，默认值为 16。调整此参数对上游压力无影响，对下游压力则影响显著。
+    worker-count: 16                 # sync 读取、解析和应用已传输 binlog event 的并发线程数量，默认值为 16。调整此参数对上游无影响，对下游则造成显著压力。
```
Comment (sunzhaoyang) [inline zh/task-configuration-file-full.md:110]:
```suggestion
    worker-count: 16                 # 读取、解析和应用已传输到本地的 binlog 的并发线程数量，默认值为 16。调整此参数对上游拉取日志的并发无影响，对下游则造成显著压力变化。
```

File: en/task-configuration-file-full.md [hunk 3]
```diff
@@ -27,21 +23,21 @@ The following is the task configuration file template which allows you to perfor
...
+task-mode: all                  # The task mode. Can be set to `full`(only migrates full data)/`incremental`(replicates binlogs synchronously)/`all` (replicates both full data and incremental binlogs).
+shard-mode: "pessimistic"       # The shard merge mode. Optional modes are ""/"pessimistic"/"optimistic". The "" mode is used by default which means no specific mode is required. If the task is a shard merge task, set it to the "pessimistic" mode. 
+                                # After understanding the principles and restrictions of the "optimistic" mode, you can set it to the "optimistic" mode.
 meta-schema: "dm_meta"          # The downstream database that stores the `meta` information.
 timezone: "Asia/Shanghai"       # The timezone.
 case-sensitive: false           # Determines whether the schema/table is case-sensitive.
-online-ddl: true                # Currently supports automatic processing of "gh-ost" and "pt".
+online-ddl: true                # Supports automatic processing of upstream "gh-ost" and "pt".
 online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` will be deprecated in the future, so it is recommended to use `online-ddl`.
```
Comment (GMHDBJD) [inline en/task-configuration-file-full.md:33]: has been deprecated in v2.0.6

File: en/task-configuration-file-full.md (hunk 1 above)
Comment (GMHDBJD) [inline en/task-configuration-file-full.md:115]: Not read, parse, and apply. Only apply

File: zh/task-configuration-file-full.md (hunk 2 above)
Comment (GMHDBJD) [inline zh/task-configuration-file-full.md:110]: ditto

**Review by GMHDBJD — COMMENTED** [review]
rest LGTM

File: en/task-configuration-file-full.md (hunk 3 above)
Comment (Liuxiaozhen12) [inline en/task-configuration-file-full.md:33]:
```suggestion
online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` has been deprecated in v2.0.6, so it is recommended to use `online-ddl`.
```

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: en/task-configuration-file-full.md [hunk 4]
```diff
@@ -116,8 +112,8 @@ loaders:
 # Configuration arguments of the sync processing unit.
 syncers:
   global:                            # The configuration name of the processing unit.
-    worker-count: 16                 # The number of threads that replicate binlog events concurrently in the sync processing unit. When multiple instances are migrating data to TiDB at the same time, reduce the value according to the load.
-    batch: 100                       # The number of SQL statements in a transaction batch that the sync processing unit replicates to the downstream database (100 by default).
+    worker-count: 16                 # The number of concurrent threads that apply binlogs which has been transferred to the local (16 by default). Adjusting this parameter has no effect on the concurrency of upstream pull logs, but has a significant effect on the downstream database.
```
Comment (hfxsd) [inline en/task-configuration-file-full.md:115]:
```suggestion
    worker-count: 16                 # The number of concurrent threads that apply binlogs which have been transferred to the local (16 by default). Adjusting this parameter has no effect on the concurrency of upstream pull logs, but has a significant effect on the downstream database.
```

File: zh/task-configuration-file-full.md [hunk 5]
```diff
@@ -107,7 +107,7 @@ loaders:                             # load 处理单元的运行配置参数
 
 syncers:                             # sync 处理单元的运行配置参数
   global:                            # 配置名称
-    worker-count: 16                 # sync 读取和应用已传输 binlog event 的并发线程数量，默认值为 16。调整此参数对上游压力无影响，对下游压力则影响显著。
+    worker-count: 16                 # 应用已传输到本地的 binlog 的并发线程数量，默认值为 16。调整此参数对上游拉取日志的并发无影响，对下游则造成显著压力变化。
```
Comment (hfxsd) [inline zh/task-configuration-file-full.md:110]:
```suggestion
    worker-count: 16                 # 应用已传输到本地的 binlog 的并发线程数量，默认值为 16。调整此参数不会影响上游拉取日志的并发，但会对下游产生显著压力。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**